openstack_controller/datadog_checks/openstack_controller/data/conf.yaml.example

```diff
@@ -82,38 +82,49 @@ instances:
         domain:
             id: <DOMAIN_ID>
 
-  ## @param collect_hypervisor_load - optional - boolean - default:false
+  ## @param collect_hypervisor_load - optional - boolean - default:true
   ## Collects hypervisor_load.1/5/15 for each hypervisor
   ## Disabled by default to increase performance of the check.
   ## With this enabled there is an extra N requests per check run (N = # of hypervisors)
   ## If the Agent is installed on each hypervisor, this metric is available as system.load.1/5/15
   #
-  #  collect_hypervisor_load: false
+  #  collect_hypervisor_load: true
 
+  ## @param incremental_servers_update - optional - boolean - default:true
+  ## When incremental_servers_update is set to true, instead of fetching all active servers on every check runs, it will
+  ## keep all servers in a cache and only perform incremental changes based on servers status. This is to limit the
+  ## number of HTTP calls to the nova API. This is setting can improve performances. Especially on large OpenStack
+  ## deployments with stable VMs.
+  #
+  #  incremental_servers_update: true
 
   ## @param collect_hypervisor_metrics - optional - boolean - default:true
   ## Collects hypervisor metrics (including hypervisor_load).
   #
   #  collect_hypervisor_metrics: true
 
-
   ## @param collect_project_metrics - optional - boolean - default:true
-  ## Collects project limits.
+  ## The admin user defined for Datadog starts with a "default" project.
+  ## If this option is disabled, project limits metrics are only collected from projects the
+  ## datadog defined user has access to view.
   #
   #  collect_project_metrics: true
 
-
   ## @param collect_network_metrics - optional - boolean - default:true
   ## Collects network metrics.
   #
   #  collect_network_metrics: true
 
-
-  ## @param collect_server_metrics - optional - boolean - default:true
-  ## Collects server diagnostic metrics.
+  ## @param collect_server_flavor_metrics - optional - boolean - default:true
+  ## Collect server flavor metrics
   #
-  #  collect_server_metrics: true
+  #  collect_server_flavor_metrics: true
```

Comment:
Do we want to default to this and set `collect_server_diagnostic_metrics` to false by default? Not sure if we indeed want both enabled by default

Reply:
They are a bit redundant though no? Just want to be cautious about extra expense vs benefit for default settings. If there is value in both separately I'm ok with this, just wanted to bring it up.